docs/kyma/03-01-components.md

```diff
@@ -53,7 +53,7 @@ Logging in Kyma uses [Loki](https://github.com/grafana/loki), a Prometheus-like
 
 ## Backup
```

Comment:
Let's remove it as it is not a component anymore?

Reply:
Done